### PR TITLE
Desktop version pull request

### PR DESCRIPTION
**Pull request by alicemirigo92**
Add desktop version
Add images
Add required transitions and animations
Have pairing program

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalidate* :red_circle:

Hi @alicemirigo92, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that did not make use of [new README template](https://github.com/microverseinc/readme-template). Kindly use that template to make your README professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @alicemirigo92, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed 👍🏼
- Media query ✔️ 

## Required Changes ♻️
_Check the comments under the review._

### I would have made these comments inline but they have been merged as it is a modification milestone
- [ ] Just as it is in the figma, kindly make the text align to the left but it should not pass the middle of the page.
### Your work
image

### Figma design
image
- [ ] Kindly reduce the space between your start collaboration and the text above it.
### Your work
image
### Figma design
image
- [ ] Kindly remove the spaces that are not needed by giving the webpage a width, you can give it 100% and it should work.

### Your work
image

### figma
image

- [ ] Same thing applies to this also. The width overflows

### Your work
image

### Figma
image



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — CHANGES_REQUESTED** [review]
Hi @alicemirigo92 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌

⚠️  You have submitted a project, that has failed linters, please fix them and resubmit your work



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

### Cheers and Happy coding!👏👏👏

------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

-----
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._ 🟢

File: style.css [hunk 1]
```diff
@@ -527,227 +527,318 @@ form > input {
...
+  .head-button {
+    width: 29%;
+    height: 70px;
+    left: 160px;
+    background-color: #1a2236;
+    border-radius: 0;
+    padding-left: 20px;
+    margin-top: 30px;
+    margin-right: 20px;
+  }
```
Comment (OyePriscilla) [inline style.css]: - [optional] Please kindly give this button element tect color of `#fff` as we have it in figma. 👍

**Review by OyePriscilla — APPROVED** [review]
Hi @alicemirigo92  👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters check passed ✔️ 
Good commit messages ✔️ 

## Optional suggestions
[optional] Please kindly ensure to observe `Github flow` for your subsequent PR. (Make main branch to be the default) 👍 
[optional] I noticed that you app is not responsive as expected, please take your time to work over it. 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**